Removed condescending language from Command Reference (#1522)
iterative/dvc.org: content/docs/command-reference/checkout.md

bigrams-experiment      <- Uses bigrams to improve the model
```

> We can now run `dvc checkout` that will update the most recent `model.pkl`,

Reviewer: ```suggestion
We can now run `dvc checkout` to update the most recent `model.pkl`,
```
The paragraph will need formatting after this change though.

> Same as in https://github.com/iterative/dvc.org/pull/1396/files#diff-d159337499c6716e1d89e1c12b5a8d09
